packages/compass-e2e-tests/tests/instance-sidebar.test.ts

```diff
@@ -114,6 +114,9 @@ describe('Instance sidebar', function () {
   });
 
   it('can create a database and drop it', async function () {
+    // TODO(COMPASS-7086): flaky test
+    this.retries(5);
+
```

Comment:
super tiny nit, would it make sense to add a timestamp at the end of `dbName` / `collectionName` to make sure that one run doesn't affect the other?

Reply:
Hmmm, so yes, I'll add it, but also I expect that before / after in this suite does enough to clean up even if test failed. Sometimes we don't do this thoroughly enough, but this is usually easy to spot as all the tests in the suite will start failing even if only one actually caused it, which doesn't seem to be a case here, we always see only this one failing

Reply:
Any idea what actually fails here? Wouldn't be great if users have to retry to add a database.

Reply:
We have ticket to investigate and first response supposed to take a look when possible, nobody had time yet. It flakes only on windows, only on 60x-enterprise run, and only for database test, not for collection one which is identical

Reply:
It errors on a line where it checks that the create database modal opened after the sidebar create database button was clicked. It might be some timing issue or something obscuring the button click on certain runs.